tests/src/test/scala/com/dwolla/security/crypto/TestKey.scala

```diff
@@ -157,4 +157,152 @@ object TestKey {
       |=RTQG
       |-----END PGP PRIVATE KEY BLOCK-----
       |""".stripMargin
+
+  val secondaryKey: String =
```

Comment:
@CJSmith-0141 do you happen to remember how you generated these keys? Also, what's the difference between them? (When I copy them both to my clipboard and then run `pbpaste | gpg --show-keys` it looks like I get the same output.)

Reply:
The first key is a superset of the second,  I believe. I exported the second one with `gpg --export-secret-subkeys <key id>`

Reply:
The initial generation was using the interactive prompt: `gpg --full-generate-key` -> `(1) RSA and RSA` -> `What keysize do you want? (3072)` -> `Key is valid for? (0)` -> `Real name: key 2` -> `Email address: key2@dwolla.com` -> no passphrase

Reply:
If the second is a subset of the first, do you think we need to include both? They're just used for reference, right (i.e., not used as part of the test code or anything)?

Reply:
I think it's safe to delete. My original though in adding it to the repo was to use it in a test that loads the key via `PGPKeyAlg[F].readPrivateKey()` [like the one here](https://github.com/Dwolla/fs2-pgp/blob/main/tests/src/test/scala/com/dwolla/security/crypto/CryptoAlgSpec.scala#L294).

Reply:
That makes sense. Plus it's a good idea to have the key to decrypt the cryptotext used in the test case you contributed. I'll update my PR to remove the second subkey and then we'll try to get this released ASAP.